translations/round_structure.tex/pl.po

```diff
@@ -52,7 +52,7 @@ msgstr ""
 #. type: Plain text
 #: sections/round_structure.tex:3
 msgid "\\addsection{Round Structure}{\\spells/forgetfulness.png}"
-msgstr ""
+msgstr "\\addsection{Struktura Rundy}{\\spells/forgetfulness.png}"
```

Comment:
luźna myśl

```suggestion
msgstr "\\addsection{Runda}{\\spells/forgetfulness.png}"
```

Reply:
Sama Runda jest trochę nieprecyzyjna?

Reply:
Zgadzam się z powyższym.
"Struktura rundy" może brzmi dziwnie bez kontekstu, ale jest to bardziej precyzyjny zwrot